website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -29,6 +33,608 @@ resource "newrelic_cloud_aws_link_account" "foo" {
+* `tag_value` - (Optional) Specify a Tag value associated with the resources that you want to monitor. Filter values are case-sensitive.
+  * the following integration blocks support the usage of this resource: `vpc`, `sqs`, `ebs`, `alb`, `elasticache`, `api_gateway`, `cloudfront`, `dynamodb`, `ec2`, `ecs`, `efs`, `elasticbeanstalk`, `elasticsearch`, `emr`, `iam`, `kinesis`, `lambda`, `rds`, `redshift`
+* `fetch_extended_inventory` - (Optional) Determine if extra inventory data be collected or not. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `sqs`, `ebs`, `alb`, `cloudfront`, `dynamodb`, `elasticbeanstalk`, `elb`, `route53`, `sns`
+* `fetch_tags` - (Optional) Specify if tags should be collected. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `sqs`, `alb`, `elasticache`, `cloudfront`, `dynamodb`, `ecs`, `efs`, `elasticbeanstalk`, `elasticsearch`, `emr`, `kinesis`, `lambda`, `rds`
+* `queue_prefixes` - (Optional) Specify each name or prefix for the Queues that you want to monitor. Filter values are case-sensitive.
+  * the following integration blocks support the usage of this resource: `sqs`
+* `load_balancer_prefixes` - (Optional) Specify each name or prefix for the LBs that you want to monitor. Filter values are case-sensitive.
+  * the following integration blocks support the usage of this resource: `alb`
+* `stage_prefixes` - (Optional) Determine if extra inventory data be collected or not. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `api_gateway`
+* `fetch_lambdas_at_edge` - (Optional) Specify if Lambdas@Edge should be monitored. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `cloudfront`
+* `duplicate_ec2_tags` - (Optional) Specify if the old legacy metadata and tag names have to be kept, it will consume more ingest data size.
+  * the following integration blocks support the usage of this resource: `ec2`
+* `fetch_ip_addresses` - (Optional) Specify if IP addresses of ec2 instance should be collected.
+  * the following integration blocks support the usage of this resource: `ec2`
+* `fetch_shards` - (Optional) Specify if Shards should be monitored. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `kinesis`
+
+
+All of these arguments may be used only with a certain set of arguments. In order to find the right set of arguments which go with each integration **check out the [Integration blocks](#integration-blocks) section below.**
```

Comment:
It might make sense to shift this line to the top, under `account_id` and `linked_account_id` under a `h4` "Arguments to be Specified with Integration Blocks" to make it look like

#### Arguments to be Specified with Integration Blocks

The following arguments are intended to be used within certain "integration blocks" in the resource, i.e. these are supposed to be specified as nested arguments "within" an integration, unlike `account_id` and `linked_account_id` which are specified at the resource level. An exhaustive list of all of such arguments supported by the resource (and the integration blocks they would need to be specified with) has been given below.

In order to find the right set of arguments which go with each integration, **check out the [Integration Blocks](#integration-blocks) section below.**